.github/workflows/ci-notebooks.yml

```diff
@@ -11,6 +11,9 @@ jobs:
   test-notebooks:
```

Comment:
```suggestion
  tutorial-notebooks:
```

Reply:
Why wasn't this applied?

Reply:
It wasn't applied by mistake, so it is applied now.